src/test/scala/cognite/spark/v1/FlexibleDataModelsRelationTest.scala

```diff
@@ -350,16 +430,8 @@ class FlexibleDataModelsRelationTest extends FlatSpec with Matchers with SparkTe
     readDf.createTempView(s"instance_filter_table")
 
     val sql = s"""
-                 |select forEqualsFilter as externalId from instance_filter_table
+                 |select * from instance_filter_table
                  |where
-                 |forEqualsFilter = 'str1' and
-                 |forInFilter in ('str1', 'str2', 'str3') and
-                 |forGteFilter >= 1 and
-                 |forGtFilter > 1 and
-                 |forLteFilter <= 2 and
-                 |forLtFilter < 4 and
-                 |(forOrFilter1 == 5.1 or forOrFilter2 == 6.1) and
-                 |forIsNotNullFilter is not null and
                  |forIsNullFilter is null
```

Comment:
I might miss it but do we have a replacement somewhere to test other filters instead of only is null? The old test case looks better with different filter condition and values (even though test is still ignore though)

Reply:
This test is ignored. 
FDM still haven't released the fix.

Reply:
Yes but later please add back some tests for better filter condition rather than just isNull :)